content/docs/03-cluster-profiles/3-cluster-profile-import-export.md

```diff
@@ -0,0 +1,102 @@
+---
+title: "Import Export Cluster Profiles"
+metaTitle: "Import and Export Palette Cluster Profiles"
+metaDescription: "The method for importing and exporting Cluster Profile on Spectro Cloud"
+icon: ""
+hideToC: true
+fullWidth: false
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+
+
+# Overview
+Palette enables cluster profiles to be exported and then imported across multiple environments, projects and tenants. This smoothens the reuse and sharing of huge profiles with large number of add-ons and integrations. 
+
+# Prerequisites
+
+* [Export](/cluster-profiles/cluster-profile-import-export#exportclusterprofile) the cluster profile file in JSON format from the Palette console.
+
+
+* The packs getting imported as part of the profile need to be present at the destination repositories.
+
+
+* The `macros` needs to be present within the destination for importing a cluster profile. If not [create the macros](/clusters/cluster-management/macros#createyourmacro) at the destination.
+
```

Comment:
Should we say like `The macros used in the exported profile should be available in the target environment during import. If not [create the macros](/clusters/cluster-management/macros#createyourmacro) at the target environment.` 

Instead of `destination`, we can say `target environment`